final-progII/src/presentation/Screen.java

```diff
@@ -439,7 +440,7 @@ public void actionPerformed(ActionEvent e) {
 					controller.createUser(functionary);
 				} catch (Exception e1) {
 					// TODO Auto-generated catch block
-					e1.printStackTrace();
+					JOptionPane.showMessageDialog(null, " The CI already exits");
```

Comment:
Could this trigger an alert for every exception?

Reply:
No